packages/app/src/components/time-series-chart/logic/common.ts

```diff
@@ -15,26 +14,29 @@ export interface BenchmarkConfig {
 
 export type TimespanAnnotationConfig =
   | {
-      type: 'bar';
+      type: 'solid';
```

Comment:
I think we can get rid of the type property and have a single timespan annotation type with a `fill` prop
```suggestion
      fill?: 'solid' | 'hatched'; // optional so the default can be `solid`
```

Reply:
And what if there's a third type next week?

Reply:
That's not really an argument 😛  we've been working with a single type for several months, also a solid/hatched fill makes sense configuration-wise imho, but do what feels best.

Reply:
We've also been working with a domain that consists of two values for several months, yet you insist on checking for larger domains...
But, if it helps you sleep at night, I'll change this to the fill prop solution. I'm getting a bit tired of the back and forth here.

Reply:
> We've also been working with a domain that consists of two values for several months, yet you insist on checking for larger domains..

that's because I read that in d3 documentation last week ;-)